### PR TITLE
ROCM: fixed coverity issue

### PR DESCRIPTION
**Pull request by hoopoepg**

### CODE REVIEW

File: src/uct/rocm/cma/rocm_common.c [hunk 1]
```diff
@@ -207,7 +207,11 @@ static hsa_status_t uct_rocm_check_ptr_info(void *ptr, void **gpu_ptr,
     info.size = sizeof(hsa_amd_pointer_info_t);
 
     *need_lock = 1;
-    *gpu_ptr = 0;
+
```
Comment (evgeny-leksikov) [inline src/uct/rocm/cma/rocm_common.c]: need to add mellanox copyright

**Comment by yosefe** [comment]
@yuq pls take a look

File: src/uct/rocm/cma/rocm_common.c (hunk 1 above)
Comment (hoopoepg) [reply src/uct/rocm/cma/rocm_common.c]: fixed

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9019/ for details (Mellanox internal link).

File: src/uct/rocm/cma/rocm_common.c [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) Advanced Micro Devices, Inc. 2016 - 2017. ALL RIGHTS RESERVED.
+ * Copyright (C) Mellanox Technologies Ltd. 2019.       ALL RIGHTS RESERVED.
```
Comment (evgeny-leksikov) [inline src/uct/rocm/cma/rocm_common.c]: Why so many whitespaces? In all files we have only 2 :)

Comment (hoopoepg) [reply src/uct/rocm/cma/rocm_common.c]: single year, later will add 2019-....

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9020/ for details (Mellanox internal link).

File: src/uct/rocm/cma/rocm_common.c (hunk 2 above)
Comment (evgeny-leksikov) [reply src/uct/rocm/cma/rocm_common.c]: anyway it looks strange

File: src/uct/rocm/cma/rocm_common.c (hunk 1 above)
Comment (yuq) [reply src/uct/rocm/cma/rocm_common.c]: All caller of this function will set gpu_ptr, so this is just a warning of coverity, no real seg fault, right?

Comment (yosefe) [reply src/uct/rocm/cma/rocm_common.c]: in this case, why is gpu_ptr checked for NULL later in this function?

Comment (hoopoepg) [reply src/uct/rocm/cma/rocm_common.c]: few lines below there is evaluation for pointer

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9024/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
@yosefe ok to merge? failures are not related to fix (doc build failed, build timeout and DC failed on cuda)

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9041/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
@yosefe ok to merge?

**Pull request merged**